.buildkite/scripts/common.sh

```diff
@@ -94,6 +94,33 @@ upload_packages_to_gcp_bucket() {
     done
 }
 
+upload_mbp_packages_to_gcp_bucket() {
+    pattern=${1}
+    type=${2}
+    baseUri="gs://${JOB_GCS_BUCKET}/jobs/buildkite"              #TODO: needs to delete the "/buildkite" part after the migration from Jenkins
+
+    if [[ ${type} == "snapshot" ]]; then
+        bucketUri="${baseUri}"/commits/${BUILDKITE_COMMIT}
+    else
+        bucketUri="${baseUri}"/${type}/${BUILDKITE_COMMIT}
+    fi
+    gsutil -m -q cp -a public-read -r ${pattern} "${bucketUri}"
```

Comment:
nit, I wold use `local` when declaring variables inside a function

```suggestion
    local pattern=${1}
    local type=${2}
    local baseUri="gs://${JOB_GCS_BUCKET}/jobs/buildkite"              #TODO: needs to delete the "/buildkite" part after the migration from Jenkins
    local bucketUri=""

    if [[ ${type} == "snapshot" ]]; then
        bucketUri="${baseUri}"/commits/${BUILDKITE_COMMIT}
    else
        bucketUri="${baseUri}"/${type}/${BUILDKITE_COMMIT}
    fi
    gsutil -m -q cp -a public-read -r ${pattern} "${bucketUri}"
```